### ISSUE TITLE
UploadThingError: Missing UPLOADTHING_SECRET env variable when calling deleteFiles()

### ISSUE THREAD

**Issue by Caiocarvalhodigital** [pingdotgg/uploadthing#412]
### Steps to reproduce:
Set up an API endpoint using the utapi module to delete files:

```
import { utapi } from "server/uploadthing";
import type { NextApiResponse, NextApiRequest } from "next";

export default async function handler(
  req: NextApiRequest,
  res: NextApiResponse
) {
  try {
    const { fileKey } = req.query;
    if (fileKey) {
      console.log("fileKey: ", fileKey);
      const response = await utapi.deleteFiles(fileKey);
      console.log("response: ", response);
      return res.status(200).json(response);
    }
  } catch (e) {
    console.error("Request error", e);
    res.status(500).json({ error: "Error deleting file" });
  }
}
```

Set the UPLOADTHING_SECRET and UPLOADTHING_APP_ID environment variables.
Call the API endpoint with a valid fileKey:
```
await fetch(
          `${process.env.NEXT_PUBLIC_URL}/api/uploadThing/delete?fileKey=${filenameWithExtension}`
        );
```

My server/uploadthing.ts looks like this:

```
import type { NextApiRequest, NextApiResponse } from "next";
import { createUploadthing, type FileRouter } from "uploadthing/next-legacy";
import { UTApi } from "uploadthing/server";
const f = createUploadthing();
const auth = (req: NextApiRequest, res: NextApiResponse) => ({ id: "fakeId" }); // Fake auth function
// FileRouter for your app, can contain multiple FileRoutes
export const ourFileRouter = {
  // Define as many FileRoutes as you like, each with a unique routeSlug
  imageUploader: f({ image: { maxFileSize: "1MB" } })
    // Set permissions and file types for this FileRoute
    .middleware(async ({ req, res }) => {
      // This code runs on your server before upload
      const user = await auth(req, res);
      // If you throw, the user will not be able to upload
      if (!user) throw new Error("Unauthorized");
      // Whatever is returned here is accessible in onUploadComplete as `metadata`
      return { userId: user.id };
    })
    .onUploadComplete(async ({ metadata, file }) => {
      // This code RUNS ON YOUR SERVER after upload
      console.log("Upload complete for userId:", metadata.userId);
      console.log("file url", file.url);
    }),
} satisfies FileRouter;
export type OurFileRouter = typeof ourFileRouter;
export const utapi = new UTApi({
  fetch: globalThis.fetch,
  apiKey: process.env.UPLOADTHING_SECRET,
});
```

### Expected behavior:
The file should be deleted successfully.

### Actual behavior:
An error is thrown: UploadThingError: Missing UPLOADTHING_SECRET env variable.

### Environment:
"next": "13.2.4",
"react": "18.2.0",
"uploadthing": "5.7"
"@uploadthing/react": "^5.6.2",
Node v18.16.0

### Error:
Request error UploadThingError: Missing UPLOADTHING_SECRET env variable.
at getApiKeyOrThrow (file:///.../node_modules/uploadthing/dist/chunk-5APOLRUY.mjs:40:9)
at UTApi.requestUploadThing (file:///.../node_modules/uploadthing/dist/chunk-5APOLRUY.mjs:151:5)
at UTApi.deleteFiles (file:///.../node_modules/uploadthing/dist/chunk-5APOLRUY.mjs:260:17)
at handler (webpack-internal:///(api)/./pages/api/uploadThing/delete/index.ts:15:90)
at Object.apiResolver (/.../node_modules/next/dist/server/api-utils/node.js:372:15)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
at async DevServer.runApi (/.../node_modules/next/dist/server/next-server.js:514:9)
at async Object.fn (/.../node_modules/next/dist/server/next-server.js:828:35)
at async Router.execute (/.../node_modules/next/dist/server/router.js:243:32)
at async DevServer.runImpl (/.../node_modules/next/dist/server/base-server.js:432:29)
at async DevServer.run (/.../node_modules/next/dist/server/dev/next-dev-server.js:831:20)
at async DevServer.handleRequestImpl (/.../node_modules/next/dist/server/base-server.js:375:20)
at async /.../node_modules/next/dist/server/base-server.js:157:99 {
code: 'MISSING_ENV',
data: undefined,
cause: undefined
}

I hope this information is helpful. Please let me know if you have any other questions.

**Comment by juliusmarminge** [comment]
Fixed in 5.7.1

**Comment by iamnevir** [comment]
i still have this issue
